### PR TITLE
Newsletter 13: Add Minigene and Event Chaining

### PR DESCRIPTION
**Pull request by jojolepro**
From https://github.com/rust-gamedev/rust-gamedev.github.io/issues/232
:)

### CODE REVIEW

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -222,8 +222,42 @@ organized with one branch for each subsection.
...
 ## Library & Tooling Updates
 
+### [Minigene][minigene]
+
+[Minigene][minigene] is a tiled and ascii game engine made by [@jojolepro].
+It allows to very simply create complex games running on desktop as well as
+in the browser.
+
+While it is still under heavy development, a lot can be done already:
+- Easily Create ECS Systems.
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:253]:
```suggestion

- Easily Create ECS Systems.
```

File: content/posts/newsletter-013/index.md [hunk 2]
```diff
@@ -222,8 +222,42 @@ organized with one branch for each subsection.
...
 [@stevebob]: https://github.com/stevebob
 
+### [Event Chaining as a Decoupling Method in ECS][event-chaining]
+
+[This blog post][event-chaining] by [@jojolepro] is an in-depth look at how
+using events in entity-component-system architectures can improve
+system reusability dramatically.
+
+Using events in this way also allows for:
+- Easier Testing
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:232]:
```suggestion

- Easier Testing
```

**Review by 17cupsofcoffee — COMMENTED** [review]
Fixes for the CI errors:

**Comment by jojolepro** [comment]
Thanks!

**Review by ozkriff — APPROVED** [review]
:+1: (made a few style tweaks & added an image)

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**